<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" ignored="0" total="1" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="40952" started-at="2018-11-16T19:54:31Z" finished-at="2018-11-16T19:55:11Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="40952" started-at="2018-11-16T19:54:31Z" finished-at="2018-11-16T19:55:11Z">
      <class name="ExceptionHandlingDemo.ExceptionHandlingWithSeleniumDemo">
        <test-method status="PASS" signature="testCase1()[pri:0, instance:ExceptionHandlingDemo.ExceptionHandlingWithSeleniumDemo@74650e52]" name="testCase1" duration-ms="40905" started-at="2018-11-16T19:54:31Z" finished-at="2018-11-16T19:55:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCase1 -->
      </class> <!-- ExceptionHandlingDemo.ExceptionHandlingWithSeleniumDemo -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
